rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/indices.rollover/10_basic.yml

```diff
@@ -183,3 +183,11 @@
             min_age: "0s"
             min_docs: 1
   - match: { error.reason: "Validation Failed: 1: at least one max_* rollover condition must be set when using min_* conditions;" }
+
+---
+"Rolling over an unknown target should return 404":
+  - do:
+      catch: bad_request
+      indices.rollover:
+        alias: "non_existent"
+  - match: {error.reason: "rollover target [non_existent] does not exist"}
```

Comment:
I'm still a little bit on the fence whether I think this should be a 404 or not. I'm inclined to say it should be, but I reverted back to the old behavior for consistency. Does anyone have any strong opinions/insights?

Reply:
It feels like a 404 is probably the right response to me.